R/zzz-methods.R

```diff
@@ -26,7 +26,12 @@ on_load({
 #' tibble(a = 1:3) %>%
 #'   mutate(b = a + 1)
 methods_overwrite <- function() {
-  cli::cli_inform(c(i = "Overwriting {.pkg dplyr} methods with {.pkg duckplyr} methods"))
+  cli::cli_inform(
+    c(
+      v = "Overwriting {.pkg dplyr} methods with {.pkg duckplyr} methods",
+      i = "Turn off with {.run duckplyr::methods_restore()}"
```

Comment:
```suggestion
      v = "Overwriting {.pkg dplyr} methods with {.pkg duckplyr} methods.",
      i = "Turn off with {.run duckplyr::methods_restore()}."
```

We do at least make an attempt at using full stops at the end of all our bullets